docusaurus/docs/recipes/favicon.md

```diff
@@ -0,0 +1,34 @@
+---
+id: recipe-add-favicon-manifest
+title: Generate favicons & manifest
+sidebar_label: Generate favicons & manifest
+---
+
+In this recipe, you will be guided through the process of setting up a favicon for your app that is supported by the latest versions of the [browsers](https://browserl.ist/?q=last%202%20Chrome%20versions%2C%20last%202%20Firefox%20versions%2C%20last%202%20Safari%20versions%2C%20last%202%20Edge%20versions%2C%20last%202%20ChromeAndroid%20versions%2C%20last%202%20iOS%20versions) this boilerplate supports.
+
+## Walk-through
+
+### 1. Use a favicon generator
+
+It is recommended the use of the generator [RealFaviconGenerator](https://realfavicongenerator.net/). Follow the next steps:
+
+- Select an image (PNG, JPG or SVG). Your image should be 260x260 or more for optimal results.
```

Comment:
Can you use a ordered list here (numbers) just like we are using in private recipes?

Reply:
✅